frontend/.env.sample

```diff
@@ -1,7 +1,4 @@
-VITE_BACKEND_HOST="127.0.0.1:3000"
-VITE_USE_TLS="false"
-VITE_INSECURE_SKIP_VERIFY="false"
-VITE_FRONTEND_PORT="3001"
+VITE_BACKEND_BASE_URL="localhost:3000"
```

Comment:
Would it make sense to add a comment like `# Base URL without protocol` or similar?

Reply:
Done